docs/podman-image-tree.1.md

```diff
@@ -0,0 +1,74 @@
+% podman-image-tree(1)
+
+## NAME
+podman\-image\-tree - Show the image hierarchy in local image cache
+
+## SYNOPSIS
+**podman image tree** [*image*:*tag*]**|**[*image-id*]
+[**--help**|**-h**]
+
+## DESCRIPTION
+Prints layer hierarchy of an image in a tree format
+If you do not provide a *tag*, podman will default to `latest` for the *image*.
+
+## OPTIONS
+
+**--help**, **-h**
+
+Print usage statement
+
+**--whatrequires**
+
+Show all child images in tree structure
+
+## EXAMPLES
+
+```
+$ podman pull docker.io/library/wordpress
+$ podman  pull docker.io/library/php:7.2-apache
+
+$ podman image tree docker.io/library/wordpress
+Image ID     : 6e880d17852f
+RepoTags     : [docker.io/library/wordpress:latest]
+Size         : 429.8MB
+Image Layers :
+└─ ID: 3c816b4ead84 Size: 58.47MB Tags: []
+└─ ID: e39dad2af72e Size: 3.584kB Tags: []
+└─ ID: b2d6a702383c Size: 213.6MB Tags: []
+└─ ID: 94609408badd Size: 3.584kB Tags: []
+└─ ID: f4dddbf86725 Size: 43.04MB Tags: []
+└─ ID: 8f695df43a4c Size: 11.78kB Tags: []
+└─ ID: c29d67bf8461 Size: 9.728kB Tags: []
+└─ ID: 23f4315918f8 Size: 7.68kB Tags: []
+└─ ID: d082f93a18b3 Size: 13.51MB Tags: []
+└─ ID: 7ea8bedcac69 Size: 4.096kB Tags: []
+└─ ID: dc3bbf7b3dc0 Size: 57.53MB Tags: []
+└─ ID: fdbbc6404531 Size: 11.78kB Tags: []
+└─ ID: 8d24785437c6 Size: 4.608kB Tags: []
+└─ ID: 80715f9e8880 Size: 4.608kB Tags: [docker.io/library/php:7.2-apache]
+ └─ ID: c93cbcd6437e Size: 3.573MB Tags: []
+ └─ ID: dece674f3cd1 Size: 4.608kB Tags: []
+ └─ ID: 834f4497afda Size: 7.168kB Tags: []
+ └─ ID: bfe2ce1263f8 Size: 40.06MB Tags: []
+ └─ ID: 748e99b214cf Size: 11.78kB Tags: [docker.io/library/wordpress:latest]
+
+$ podman image tree e2834342b873 --whatrequires
+Image ID     : e2834342b873
+RepoTags     : [docker.io/library/php:7.2-apache]
+Size         : 43.66MB
+Image Layers :
+└─ ID: 80715f9e8880 Size: 4.608kB Tags: [docker.io/library/php:7.2-apache]
+ └─ ID: c93cbcd6437e Size: 3.573MB Tags: []
+ └─ ID: dece674f3cd1 Size: 4.608kB Tags: []
+ └─ ID: 834f4497afda Size: 7.168kB Tags: []
+ └─ ID: bfe2ce1263f8 Size: 40.06MB Tags: []
+ └─ ID: 748e99b214cf Size: 11.78kB Tags: [docker.io/library/wordpress:latest]
+
```

Comment:
Once the output is stable, we need to update the manpage as well.

Reply:
It may be worth mentioning that the `Top Layer of` will only be printed when the tag is known locally. Nothing will be fetched from a registry. Just to make sure that all users are aware of that.

Reply:
This is still valid.

Reply:
I mentioned that here https://github.com/containers/libpod/pull/1642/files#diff-1fe835d4efa3abb9d3d5535cb63ed37dR13